actions/user.js

```diff
@@ -0,0 +1,8 @@
+import * as types from '../constants/actionTypes';
+
+export const setCurrentUser = (user, userToken) => {
+    return {
+        type: types.SET_CURRENT_USER,
```

Comment:
not using this for now... not sure if we need it yet. just moved it here for now.